cookbooks/bcpc-hadoop/recipes/oozie.rb

```diff
@@ -218,6 +218,23 @@
       Digest::MD5.hexdigest(File.read(OOZIE_SHARELIB_TARBALL_PATH))
   end
   action :nothing
+
+  
+  notifies :run, 'ruby_block[oozie sharelib sqoop-action workaround for 2.6.1]', :immediately
+end
+
+ruby_block 'oozie sharelib sqoop-action workaround for 2.6.1' do
+  block do
+    node[:bcpc][:hadoop][:oozie_hosts].each do |oozie_host|
+      instrumentation = shell_out! "#{OOZIE_CLIENT_PATH}/bin/oozie admin "\
```

Comment:
Doesn't work on a repxed Oozie head node since there is no Oozie service running yet.

Reply:
This might explain why in some repxe of Oozie heads the sharelib was not upgraded and was simply silently skipped: https://github.com/bloomberg/chef-bach/blob/master/cookbooks/bcpc-hadoop/libraries/utils.rb#L458-L460

Reply:
But if Oozie is in HA, the sharelib update will be successful as it will be able to connect to the other Oozie machines (old or new versions)